content/streamlit-cloud/get-started/manage-your-app.md

```diff
@@ -176,3 +177,82 @@ To unfavorite an app, click the star (⭐) again.
 Click [here](/knowledge-base/deploy/upgrade-streamlit-version-on-streamlit-cloud) to learn more about upgrading the Streamlit version of your app on Streamlit Cloud.
 
 </Tip>
+
+## Analytics Modal
+
+Once you start or are invited to a Streamlit workspace, you have access to 2 types of analytics:
+
+1. [Workspace analytics](/streamlit-cloud/get-started/manage-your-app#workspace-analytics): shows you how many viewers in total have visited _all_ the apps in your workspace.
+   <Image alt="Workspace analytics" src="/images/streamlit-cloud/workspace-analytics.gif" />
+2. [App viewers](/streamlit-cloud/get-started/manage-your-app#app-viewers): shows you who has recently viewed your workspace’s individual apps and when.
+   <Image alt="Workspace analytics" src="/images/streamlit-cloud/app-viewers-data.gif" />
+
+<Note>
+
+The Analytics Modal is visible to everyone with access to your workspace, including admins, developers, or anyone with viewer access to a workspace.
+
+</Note>
+
+### Workspace analytics
+
+Streamlit Cloud enables you to view analytics data for all apps in your workspace in one central dashboard. At a glance, you get an overview of how active your workspace is and how popular your apps are.
+
+Say you're the Head of Data and want to see if the Streamlit usage in your company has increased over time. Or say you're an open-source developer and want to see how popular your apps have become. The Workspace analytics dashboard is your one-stop-shop for these insights.
+
+To view your Workspace analytics:
+
+1. Select the "**Analytics**" option on the dashboard header
+   ![Workspace analytics header dashboard](/images/streamlit-cloud/workspace-analytics-header.png)
+2. View the "**Workspace**" tab in the Analytics modal
+   ![Workspace analytics modal](/images/streamlit-cloud/workspace-analytics-modal.png)
+
+You're presented with a graph that you can hover over to see the number of users who have viewed at least one app in your workspace that month. This viewers count includes apps that anyone in your workspace created.
+
+Solid lines indicate fully-complete months on the dashboard, while dotted lines indicate the current in-progress month.
+
+**Note:** Viewers data on your dashboard starts from April 2022 and onward. April 2022 data was our first month comprehensively tracking user analytics in Streamlit workspaces, and our tracking is even more refined starting in May 2022 and onward.
+
+### App viewers
+
+In addition to a general overview of the activity of your workspace and the popularity of your apps, Streamlit Cloud allows you to drill down to the level of individual apps and understand their viewership better.
+
+As an app developer or a viewer with access to a given workspace, you can see who has viewed a given app and when. Specifically, you can see the total viewers count of your app (since April 2022 and onward), the most recent unique viewers (capped up to your last 20 viewers), and a relative timestamp of their last view.
```

Comment:
Same comment as above...if anyone with access to a workspace can view this, then it doesn't much matter to mention that developers or viewers can view statistics

Reply:
Do you not think we should disambiguate what is meant by "anyone"? If a user has access to a workspace, that user is either a developer or a viewer. I think we should call that out at least once in the doc (?)